madrasthemes/mas-elementor
# Fix/carousel attributes control

###Steps to test the issue

1. Add products widgets and select any mas-post
2. Select Enable carousel option and enable space between option.
3. Check if the space between the  products works
